### ISSUE TITLE
Feature Request: Adobe Premiere Pro

### ISSUE THREAD

**Issue by oceanscape** [lra/mackup#1745]
Is it possible to add Premiere Pro settings to Mackup?

**Comment by joshmedeski** [comment]
Yes, if you know where the user configuration files are stored a new application configuration file can be added.
